src/servicebus/azext_servicebus/__init__.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+
+# pylint: disable=unused-import
+
+
```

Comment:
Load the _helps file here otherwise it won't be loaded and so no help is shown.

e.g. https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/__init__.py#L8

Reply:
Resolved